### PR TITLE
update docs to match new schema

### PR DESCRIPTION
**Pull request by jwld**
relates #73

### CODE REVIEW

File: docs.md [hunk 1]
```diff
@@ -21,21 +21,28 @@ Status: 200 OK
...
-    "_id": "5979b779ed693f0011b17b57",
+    "_id": "597b363ae05960001124583b",
+    "updatedAt": "2017-07-28T13:03:54.818Z",
+    "createdAt": "2017-07-28T13:03:54.818Z",
+    "ownerId": "8496873ea34958810182138c",
+    "category": [
+      "music",
+      "dining"
+    ],
+    "location": "9348293df12398123885930a",
```
Comment (des-des) [inline docs.md]: whats this?

File: docs.md [hunk 2]
```diff
@@ -49,21 +56,28 @@ Status: 200 OK
...
-  "_id": "5979b779ed693f0011b17b57",
+  "_id": "597b363ae05960001124583b",
+  "updatedAt": "2017-07-28T13:03:54.818Z",
+  "createdAt": "2017-07-28T13:03:54.818Z",
+  "ownerId": "8496873ea34958810182138c",
+  "category": [
+    "music",
+    "dining"
+  ],
+  "location": "9348293df12398123885930a",
```
Comment (des-des) [inline docs.md]: ?

File: docs.md [hunk 3]
```diff
@@ -74,36 +88,45 @@ Status: 200 OK
...
```
Comment (des-des) [inline docs.md]: ?

File: docs.md [hunk 4]
```diff
@@ -114,36 +137,45 @@ Status: 201 Created
...
+  "__v": 1,
+  "_id": "597b363ae05960001124583b",
+  "updatedAt": "2017-07-28T13:03:54.818Z",
+  "createdAt": "2017-07-28T13:03:54.818Z",
+  "ownerId": "8496873ea34958810182138c",
+  "category": [
+    "music",
+    "dining"
+  ],
+  "location": "9348293df12398123885930a",
```
Comment (des-des) [inline docs.md]: ?

**Review by des-des — CHANGES_REQUESTED** [review]

**Comment by des-des** [comment]
@JWLD yo can you jus assign me when ready.

If you want others to look at before I merge then leave me out till they have reviewed.

If you explicitly want me to look at a WIP branch then drop a comment into the pull, @ me and assign to me also.

**Review by mattlub — CHANGES_REQUESTED** [review]
changes:
- fix the title (add extra line after, add Nazareth to title)
- link to the docs in the main README of the repo
- add the base url (i.e. the heroku link) to the documentation

**Comment by mattlub** [comment]
also- the multi-language thing is not addressed:

- we need maybe an example request body?

- but also the table of inputs needs to be changed.

**Comment by mattlub** [comment]
Link from the front page read me to the docs?

**Comment by jwld** [comment]
There's a link in README.md, do you mean one of the other readmes?

**Review by mattlub — APPROVED** [review]
looks generally good, need to get it merged so I can refer Mohamed to it.

**Comment by jwld** [comment]
@des-des want to cast your eye over this?

File: docs.md [hunk 5]
```diff
@@ -9,32 +12,59 @@ An API providing data for the four apps on the Nazareth Open Tourism Platform, b
...
+**Users**
+- [Get all users](#get-all-users)
+- [Get user by id](#get-user-by-id)
+- [Create user](#create-user)
+- [Update user](#update-user)
+- [Delete user](#delete-user)
+
 ## Events
 
 ### Get all events
```
Comment (des-des) [inline docs.md:38]: Okay, so here we are missing the query params on the get requests.

Ie `GET /events?locationId={}`

File: docs.md [hunk 6]
```diff
@@ -74,35 +110,69 @@ Status: 200 OK
 
 Name | Type | Description
 ---|---|---
-name | string | **Required**. The name of the event.
-category | array of strings | **Required**. The event category / categories.
-accessibilityOptions | array of strings | Event accessibility options.
+ownerId | mongoose ObjectId | id of event owner.
```
Comment (des-des) [inline docs.md:113]: Why are we calling this a mongoose ObjectId, could we call it an id instead.Too me this seems like an implementation detail.

**Review by des-des — APPROVED** [review]

**Comment by des-des** [comment]
@JWLD okay, I think there is still some stuff we can do here, but id rather have this in master, then when we are ready, we can give the whole docs a once over.

I think this is really great (I have never been on a project with such proper docs! :heart:)

I think the main thing missing atm is query params on get requests, not sure if they are needed anywhere else

**Pull request merged**